test_runner/batch_others/test_tenant_relocation.py

```diff
@@ -217,6 +217,13 @@ def test_tenant_relocation(zenith_env_builder: ZenithEnvBuilder,
 
         tenant_pg.start()
 
+        timeline_to_detach_local_path = env.repo_dir / 'tenants' / tenant.hex / 'timelines' / timeline.hex
```

Comment:
I thought that this test was disabled because it does not handle per-tenant config. So if it works https://github.com/neondatabase/neon/issues/1555 is no longer needed

Reply:
It runs on CI even: https://app.circleci.com/pipelines/github/neondatabase/neon/5864/workflows/f1a623b2-c72b-414a-91dd-953528ebcc03/jobs/57325/steps?invite=true#step-106-4634

but I'll leave the closing up to you.